Source/Atomic/Atomic2D/ParticleEmitter2D.h

```diff
@@ -117,6 +117,15 @@ class ATOMIC_API ParticleEmitter2D : public Drawable2D
     /// Return sprite attribute.
     ResourceRef GetSpriteAttr() const;
 
+    // ATOMIC BEGIN
+
+    /// Set emission state.
+    void SetEmitting(bool enable);
+    /// Return emission state.
+    bool IsEmitting() const;
+

```

Comment:
Its good to be consistent with the other emitter, even if the names are less than optimal.